code/modules/antagonists/clock_cult/ratvarian_cyborg_modules.dm

```diff
@@ -9,10 +9,11 @@
 
 /obj/item/clock_module/Initialize()
 	..()
-	var/datum/clockcult/scripture/S = new scripture_datum
-	name = S.name
-	desc = S.desc
-	icon_state = S.button_icon_state
+	if(scripture_datum)
```

Comment:
Should this check _ever_ fail? Seems like it should have some kind of handling if it does.